charts/HTMLTimeline.tsx

```diff
@@ -52,102 +58,91 @@ export class Timeline extends React.Component<TimelineProps> {
 
     disposers!: IReactionDisposer[]
 
-    @observable startYearInput: number = 1900
-    @observable endYearInput: number = 2000
     @observable isPlaying: boolean = false
     @observable dragTarget?: string
 
     @computed get isDragging(): boolean {
         return !!this.dragTarget
     }
 
+    // Used for storing the state of the timeline.
+    // They are not suitable for direct access because start can be greater than end.
+    // Use `startYear` and `endYear` if the correct order of bounds is needed.
+    @observable startYearRaw: TimeBound = TimeBoundValue.unboundedLeft
+    @observable endYearRaw: TimeBound = TimeBoundValue.unboundedRight
+
+    @computed get startYear(): TimeBound {
```

Comment:
Nit: Can we make privates private? Mildly helpful for readability and later refactoring.

Reply:
Given that this is a React component, and they are instantiated in JSX, isn't privacy assumed for all their properties/methods?

Reply:
Ah, fair point. I am just used to working with pure TS more where explicit privates are very valuable during refactoring.